### PR TITLE
Fix slow node adding in CaaSP staging

### PR DESCRIPTION
**Pull request by kravciak**
Fix for:
https://openqa.suse.de/tests/1027053#step/stack_controller/35
https://openqa.suse.de/tests/1027368#step/install_and_reboot/21

### CODE REVIEW

**Comment by okurz** [comment]
LGTM

File: tests/casp/stack_controller.pm [hunk 1]
```diff
@@ -74,7 +74,8 @@ sub velum_bootstrap {
     if (check_screen('velum-bootstrap-accept-nodes', 3)) {
         assert_and_click 'velum-bootstrap-accept-nodes';
         # Wait until nodes are moved from pending
-        assert_screen 'velum-nodes-accepted';
+        my $nodes = get_var('STACK_SIZE') - 2;
```
Comment (asmorodskyi) [inline tests/casp/stack_controller.pm]: looks like some magic , is it random number ?

Comment (kravciak) [inline tests/casp/stack_controller.pm]: number of workers is stack_size - admin node - controller node

Comment (asmorodskyi) [inline tests/casp/stack_controller.pm]: I understand that this is probably obvious for you , but anyway I think comment explaining this will not harm

Comment (kravciak) [inline tests/casp/stack_controller.pm]: Comment added

**Pull request merged**